ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -110,7 +110,7 @@ type RayClusterStatus struct {
 	MaxWorkerReplicas int32 `json:"maxWorkerReplicas,omitempty"`
 	// LastUpdateTime indicates last update timestamp for this cluster status.
 	// +nullable
-	LastUpdateTime metav1.Time `json:"lastUpdateTime,omitempty"`
+	LastUpdateTime *metav1.Time `json:"lastUpdateTime,omitempty"`
```

Comment:
The json marshal will assign null value to metav1.Time. Because metav1.Time is actually a string in the CRD definition. Then when we use the struct, it complains cannot convert string `null` to a metav1.Time.

Reply:
..Got it, sounds annoying.